src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,34 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    private final Random random = new Random();
+    private final ColorSupplier colorSupplier = new ColorSupplier();
+
+    public Figure getRandomFigure() {
+        final String color = colorSupplier.getRandomColor();
+        final int index = random.nextInt(5);
+        final int randomLength_1 = random.nextInt(100);
+        final int randomLength_2 = random.nextInt(100);
+        final int randomLength_3 = random.nextInt(100);
```

Comment:
It's not a good practice to use numbers in variable names, and also underscore is not used for variable names in java